interface/backend/submission/evaluator/vmck.py

```diff
@@ -65,10 +65,14 @@ def evaluate(submission):
         return response.json()["id"]
 
     def update(submission):
-        response = requests.get(
-            urljoin(
-                settings.VMCK_API_URL, f"jobs/{submission.evaluator_job_id}"
+        try:
```

Comment:
Hmm...the exception is triggered (I think) when you try to convert to the json.
So you should have something like

```
try:
    json_response = response.json()
    return json_response["state"]
except Excetpion as e:
    log.debug(...)
    return "Error"